### ISSUE TITLE
Add build information (branch, commit_id) to zos version

### ISSUE THREAD

**Issue by xmonader** [threefoldtecharchive/zosctl#58]
Add build information to easily know the exact version 

Something like that 
```
./zos --version 
zos 0.1.0 (development#70e1972c6955c590eedfb0c50381d616e329d03c
```

**Comment by xmonader** [comment]
implemented here 501b7d585954615c85c0e746eadca12576becf7c
```
 ./zos --version
zos 0.1.0 (development#501b7d585954615c85c0e746eadca12576becf7c)
```

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version :
119f136b55ffa16a67688a933f4ec5cd2378b17e

# verification steps:
-  ./zos --version
```
zos 0.1.0 (development#119f136b55ffa16a67688a933f4ec5cd2378b17e)

```
